norfair_subscribers:
  converter:
    topic: "norfair/input"

norfair_publishers:
  detections:
    topic: "norfair/output"
    queue_size: 1

norfair_setup:
  distance_function: "iou_opt"
  distance_threshold: 0.5
  hit_counter_max: 15
  initialization_delay: 7.5
  pointwise_hit_counter_max: 2
  detection_threshold: 0.02
  past_detections_length: 4
